Source/Core/DolphinWX/Debugger/CodeWindow.cpp

```diff
@@ -444,41 +444,41 @@ void CCodeWindow::CreateMenu(const SConfig& core_startup_parameter, wxMenuBar* p
   wxMenu* pCoreMenu = new wxMenu;
 
   wxMenuItem* interpreter = pCoreMenu->Append(
-      IDM_INTERPRETER, _("&Interpreter core"),
+      IDM_INTERPRETER, _("&Interpreter Core"),
       _("This is necessary to get break points"
         " and stepping to work as explained in the Developer Documentation. But it can be very"
         " slow, perhaps slower than 1 fps."),
       wxITEM_CHECK);
   interpreter->Check(core_startup_parameter.iCPUCore == PowerPC::CORE_INTERPRETER);
   pCoreMenu->AppendSeparator();
 
-  pCoreMenu->Append(IDM_JIT_NO_BLOCK_LINKING, _("&JIT Block Linking off"),
+  pCoreMenu->Append(IDM_JIT_NO_BLOCK_LINKING, _("&JIT Block Linking Off"),
                     _("Provide safer execution by not linking the JIT blocks."), wxITEM_CHECK);
 
   pCoreMenu->Append(IDM_JIT_NO_BLOCK_CACHE, _("&Disable JIT Cache"),
                     _("Avoid any involuntary JIT cache clearing, this may prevent Zelda TP from "
                       "crashing.\n[This option must be selected before a game is started.]"),
                     wxITEM_CHECK);
-  pCoreMenu->Append(IDM_CLEAR_CODE_CACHE, _("&Clear JIT cache"));
+  pCoreMenu->Append(IDM_CLEAR_CODE_CACHE, _("&Clear JIT Cache"));
 
   pCoreMenu->AppendSeparator();
-  pCoreMenu->Append(IDM_LOG_INSTRUCTIONS, _("&Log JIT instruction coverage"));
-  pCoreMenu->Append(IDM_SEARCH_INSTRUCTION, _("&Search for an op"));
+  pCoreMenu->Append(IDM_LOG_INSTRUCTIONS, _("&Log JIT Instruction Coverage"));
+  pCoreMenu->Append(IDM_SEARCH_INSTRUCTION, _("&Search for an Op"));
```

Comment:
Maybe we can rename this "op" to "instruction" while we're at it? Otherwise just upper-casing it looks a little strange to me.